File: java/control_flow/switch_statement/SwitchStatement.java
class SwitchStatement {
  enum PartsOfTheDay {
    morning,
    afternoon,
    evening,
    night,
    other
  };

  public static void main(String[] args) {
    PartsOfTheDay now;

    now = PartsOfTheDay.afternoon;

    switch (now) {
      case morning:
        System.out.println("Good morning!");
        break;
      case afternoon:
        System.out.println("Good afternoon!");
        break;
      case evening:
        System.out.println("Good evening!");
        break;
      case night:
        System.out.println("Good evening!");
        break;
    }

    now = PartsOfTheDay.evening;

    switch (now) {
      case morning:
        System.out.println("Good morning!");
        break;
      case afternoon:
        System.out.println("Good afternoon!");
        break;
      case evening:
      case night:
        System.out.println("Good evening!");
        break;
    }

    now = PartsOfTheDay.other;

    switch (now) {
      case morning:
        System.out.println("Good morning!");
        break;
      case afternoon:
        System.out.println("Good afternoon!");
        break;
      case evening:
        System.out.println("Good evening!");
        break;
      case night:
        System.out.println("Good evening!");
        break;
      default:
        System.out.println("Hello!");
        break;
    }
  }
}

File: php/control_flow/switch_statement/switch_statement.php
<?php
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

$now = "afternoon";

switch ($now) {
  case "morning":
    print("Good morning!\n");
    break;
  case "afternoon":
    print("Good afternoon!\n");
    break;
  case "evening":
    print("Good evening!\n");
    break;
  case "night":
    print("Good evening!\n");
    break;
}

$now = "evening";

switch ($now) {
  case "morning":
    print("Good morning!\n");
    break;
  case "afternoon":
    print("Good afternoon!\n");
    break;
  case "evening":
  case "night":
    print("Good evening!\n");
    break;
}

$now = "other";

switch ($now) {
  case "morning":
    print("Good morning!\n");
    break;
  case "afternoon":
    print("Good afternoon!\n");
    break;
  case "evening":
    print("Good evening!\n");
    break;
  case "night":
    print("Good evening!\n");
    break;
  default:
    print("Hello!\n");
    break;
}
